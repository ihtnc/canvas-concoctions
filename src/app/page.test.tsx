import { render } from '@testing-library/react'
import Home from './page'
import { describe, expect, test } from 'vitest'

describe('Home component', () => {
  test('should render the page', () => {
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })
})