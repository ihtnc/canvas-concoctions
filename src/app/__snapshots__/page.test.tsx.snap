// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home component > should render the page 1`] = `
<div>
  <section
    class="flex text-center max-w-96"
  >
    This is a collection of various small, interactive applications / animations that served as a playground for ideas using the canvas HTML element.
  </section>
</div>
`;
